packages/menu/src/MenuItem.ts

```diff
@@ -315,6 +316,9 @@ export class MenuItem extends LikeAnchor(
             <div id="label">
                 <slot id="slot"></slot>
             </div>
+            <span class="description">
```

Comment:
The way this is leveraged in concert with the CSS makes the Menu Items each an extra 1px tall. Seems to come from it having a `1px` top margin...

Reply:
Looks like _maybe_ this could leverage `[name=description]::slotted(*)` rather than the `.description` element.